libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/route/AnnotationsRefresher.kt

```diff
@@ -84,4 +106,21 @@ internal object AnnotationsRefresher {
 
         return result
     }
+
+    private fun List<JsonElement>.toJsonArray(): JsonArray {
+        return JsonArray().also { array ->
```

Comment:
```suggestion
        return JsonArray(this.size).also { array ->
```
small improvement that optimizes unnecessary array resizes (since we know the resulting size)